### PR TITLE
Ons icon test

### PR DESCRIPTION
**Pull request by andipavllo**

### CODE REVIEW

File: core/elements/ons-icon.es6 [hunk 1]
```diff
@@ -82,6 +82,7 @@ limitations under the License.
       var size = '' + this.getAttribute('size');
       if (size.match(/^[1-5]x|lg$/)) {
         classList.push('fa-' + size);
+        this.removeAttribute('style');
```
Comment (argelius) [inline core/elements/ons-icon.es6]: Can you check that the previous value was "*px" and then just remove the `font-size` attribute?

**Comment by andipavllo** [comment]
@argelius  I fixed it

**Comment by argelius** [comment]
@andipavllo This closes #688, right?

**Comment by andipavllo** [comment]
@argelius yes, exactly

**Pull request merged**